application-services/custom/camera-management/appcamera/types.go

```diff
@@ -188,15 +188,52 @@ type GetPresetsResponse struct {
 		Name        string `json:"Name"`
 		PTZPosition struct {
 			PanTilt struct {
-				Space string `json:"Space"`
-				X     int    `json:"X"`
-				Y     int    `json:"Y"`
+				Space string  `json:"Space"`
+				X     float64 `json:"X"`
+				Y     float64 `json:"Y"`
 			} `json:"PanTilt"`
 		} `json:"PTZPosition"`
 		Token string `json:"Token"`
 	} `json:"Preset"`
 }
 
+type GetPTZConfigurationsResponse struct {
+	PTZConfiguration []struct {
+		DefaultAbsolutePantTiltPositionSpace  string `json:"DefaultAbsolutePantTiltPositionSpace"`
```

Comment:
should this be 'DefaultAbsolutePanTiltPositionSpace'?

Reply:
Its a typo from the onvif specification end - https://www.onvif.org/ver20/ptz/wsdl/ptz.wsdl

Reply:
yeah there are actually a decent amount of typos in the specs. its suprising.